extensions/patterns/configure_ec2/exec_env/execution-environment.yml

```diff
@@ -22,8 +22,8 @@ dependencies:
 
 images:
   base_image:
-    name: docker.io/redhat/ubi9:latest
+    name: quay.io/centos/centos:stream9
```

Comment:
Why not use the ubi image from registry.access.redhat.com?

Reply:
Oh, yeah if we can get it without authenticating we should do that.